bdd/features/tipo8/_transaction.feature

```diff
@@ -0,0 +1,31 @@
+Feature: A transaction is generated, authorized and confirmed
+
+  Background:
+    Given the initiative is "Scontoditipo8"
+    And the merchant is qualified
+    And the citizen A is onboarded
+    And the citizen B is onboarded
+
+  @transaction
+  @Scontoditipo8
+  Scenario: Transaction  of amount 1115 cents authorized by the citizen
+
+  @transaction
+  @Scontoditipo8
+  Scenario: Transaction  amount 1114 cents is not authorized by the citizen
+
+  @transaction
+  @Scontoditipo8
+  Scenario: Transaction amount 10001 cents is not authorized by the citizen
+
+  @transaction
+  @Scontoditipo8
+  Scenario: After two transactions that partially erode the citizen’s budget, the third operation is rewarded with the remaining budget
```

Comment:
This scenario cannot be tested because the date of transaction creation and related citizen authorization cannot be changed because it is set within the backend. 
In this case, given the daily spending limit, it is not possible to create 3 transactions that exceed the budget.